python/addBTV.py

```diff
@@ -6,7 +6,7 @@
 from PhysicsTools.PatAlgos.tools.helpers import addToProcessAndTask, getPatAlgosToolsTask
 
 
-def update_jets_AK4(process):
+def update_jets_AK4(process, add_DeepJet):
```

Comment:
I would shuffle to logic around a bit here. Have the DeepCSV/DeepJet output nodes in as the default (~10 branches anyway) and keep the DeepCSV/DeepJet inputs as toggles

Reply:
Good idea. And would you prefer one individual toggle per type of tagger or use only the `keepInputs` toggle for enabling all tagger inputs at the same time? Don't know if there are use cases where one only needs e.g. DeepCSV but not DeepJet, or DDX...

Reply:
I am thinking as we move on we might want to be dropping DeepCSV, also not all studies will need all taggers. I think my preference would be passing a list with up to `['DeepCSV', 'DeepJet', 'DDX']` and pattern match it after, but having it as 3 bools is not that bad either

Reply:
Ok, so that would make 2^3=8 possible combinations to be added in `pfnano_cff.py` for each MC or Data configuration, right?

Reply:
In case you go with bools, I'd say it's not necessary that all combinations are part of the cff, can be added as/if needed